src/object/authenticate.c

```diff
@@ -6283,6 +6300,21 @@ au_user_name (void)
     {
       int save;
 
+      /*
+       * To reduce unnecessary code execution,
+       * the current schema name can be used instead of the current user name.
+       * 
+       * Au_user_name cannot be used because it does not always store the current user name.
+       * When au_login_method () is called, Au_user_name is not changed.
+       */
+      const char *sc_name = sc_current_schema_name ();
+      char upper_sc_name[DB_MAX_USER_LENGTH];
```

Comment:
Even if the values of DB_MAX_USER_LENGTH and DB_MAX_SCHEMA_LENGTH are currently the same, it would be better to use DB_MAX_SCHEMA_LENGTH semantically.

`char upper_sc_name[DB_MAX_SCHEMA_LENGTH ];`